website/content/api-docs/secret/pki.mdx

```diff
@@ -16,1178 +16,1109 @@ update your API calls accordingly.
 
 ## Table of Contents
 
-- [Read CA Certificate](#read-ca-certificate)
-- [Read CA Certificate Chain](#read-ca-certificate-chain)
-- [Read Certificate](#read-certificate)
-- [Read Raw Certificate](#read-raw-certificate)
-- [List Certificates](#list-certificates)
-- [Submit CA Information](#submit-ca-information)
-- [Read CRL Configuration](#read-crl-configuration)
-- [Set CRL Configuration](#set-crl-configuration)
-- [Read URLs](#read-urls)
-- [Set URLs](#set-urls)
-- [Read CRL](#read-crl)
-- [Rotate CRLs](#rotate-crls)
-- [Generate Intermediate](#generate-intermediate)
-- [Set Signed Intermediate](#set-signed-intermediate)
-- [Generate Certificate](#generate-certificate)
-- [Revoke Certificate](#revoke-certificate)
-- [Create/Update Role](#create-update-role)
-- [Read Role](#read-role)
-- [List Roles](#list-roles)
-- [Delete Role](#delete-role)
-- [Generate Root](#generate-root)
-- [Delete Root](#delete-root)
-- [Sign Intermediate](#sign-intermediate)
-- [Sign Self-Issued](#sign-self-issued)
-- [Sign Certificate](#sign-certificate)
-- [Sign Verbatim](#sign-verbatim)
-- [Tidy](#tidy)
-- [Tidy Status](#tidy-status)
+- [Notice About New Multi-Issuer Functionality](#notice-about-new-multi-issuer-functionality)
```

Comment:
maybe I'm doing this wrong, but none of these links seem to work?

Reply:
If you `cd website && make` it should launch a Vercel server that'll let you view it at e.g., `http://localhost:3000/api-docs/secret/pki` and the links definitely work for me. Maybe not all of them, but most of them definitely do and this one does.

I think GH's markdown rendering just doesn't do the linking the same as Vercel or maybe they don't render anchors for PR pages? Not sure.

Reply:
They work for me at this link: https://github.com/hashicorp/vault/blob/cipherboy-pki-pod-rotation-docs/website/content/api-docs/secret/pki.mdx